html/admin/server/utilities.twig

```diff
@@ -0,0 +1,101 @@
+{% extends "assets/template.twig" %}
```

Comment:
Not sure what others think, but I think this feature should go next to "change asset type" on the asset page itself, rather than in a standalone place